ruzj15-sp2

Jackoubek opened the pull request:
link to application: https://eso.vse.cz/~ruzj15/sp2/

Jackoubek committed:
ruzj15-sp2

link to application: https://eso.vse.cz/~ruzj15/sp2/

Jackoubek committed:
verze 02

Aplikace kompletně předělaná

https://eso.vse.cz/~ruzj15/sp2/

nvbach91 commented:
- jiz jednou pridane mesto by se nemelo pridavat znovu + upozornit uzivatele
- pridat moznost zobrazeni modaloveho okna po vyberu mesta v seznamu a tam zobrazit vice detailu o pocasi o danem meste
- seznam pridanych mest vcetne jejich nastaveni ukladejte do localStorage aby byly k dispozici kdyz uzivatel aplikaci znovu otevre s tim ze by byly videt aktualni udaje o pocasi ulozenych mest, 
- pokud uzivatel zada nazev mesta a zmackne Enter nebo klikne na tlaticko Zobrazit pocasi, provede se dotaz na server a zobrazi se vysledky, to se dela pomoci formulare a udalosti `submit`, nikoliv pres udalost `click` a udalost `keypress`
- elementy z html nevybirat opakovane
- pro delkarace funkci pouzivejte `const`
- pri volani ajax pouzivejte Promise, async/await, nikoliv callback style
- nabindovani funkce na element se ma delat jednou nikoliv opakovane

Jackoubek committed:
upravená verze 3.0

upravil jsem tyto body:

jiz jednou pridane mesto by se nemelo pridavat znovu + upozornit uzivatele
pridat moznost zobrazeni modaloveho okna po vyberu mesta v seznamu a tam zobrazit vice detailu o pocasi o danem meste
seznam pridanych mest vcetne jejich nastaveni ukladejte do localStorage aby byly k dispozici kdyz uzivatel aplikaci znovu otevre s tim ze by byly videt aktualni udaje o pocasi ulozenych mest,
pokud uzivatel zada nazev mesta a zmackne Enter nebo klikne na tlaticko Zobrazit pocasi, provede se dotaz na server a zobrazi se vysledky, to se dela pomoci formulare a udalosti submit, nikoliv pres udalost click a udalost keypress
elementy z html nevybirat opakovane
pro delkarace funkci pouzivejte const
pri volani ajax pouzivejte Promise, async/await, nikoliv callback style

https://eso.vse.cz/~ruzj15/sp2/

nvbach91 commented:
- indikace nacitani dat, disable button, loading spinner
- zvyraznit aktualni jednotku teploty

Jackoubek committed:
Finalni verze - SP2

https://eso.vse.cz/~ruzj15/sp2/
